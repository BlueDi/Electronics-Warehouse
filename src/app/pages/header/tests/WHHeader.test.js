import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import WHHeader from '@pages/header';

describe('<WHHeader />', () => {
  test('Renders properly', () => {
    const component = renderer.create(
      <Router>
        <WHHeader />
      </Router>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
