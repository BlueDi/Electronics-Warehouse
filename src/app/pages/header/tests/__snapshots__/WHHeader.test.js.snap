// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WHHeader /> Renders properly 1`] = `
<div
  className="ui huge attached menu"
>
  <a
    className="item"
    href="/"
    onClick={[Function]}
  >
    <div
      className="ui medium header"
    >
      Warehouse
    </div>
  </a>
  <div
    className="right menu"
  >
    <div
      className="item"
      onClick={[Function]}
    >
      Signup
    </div>
    <div
      aria-expanded={false}
      className="ui item simple dropdown"
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      role="listbox"
      tabIndex={0}
    >
      <div
        aria-live="polite"
        className="text"
        role="alert"
      >
        Login
      </div>
      <i
        aria-hidden="true"
        className="dropdown icon"
        onClick={[Function]}
      />
      <div
        className="right menu transition"
      />
    </div>
  </div>
</div>
`;
